src/seedu/addressbook/data/person/PostalCode.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.person;
+
```

Comment:
Do add a comment to describe what the class does (you have done that for `Unit` and `Street` classes).